skbeam/core/mask.py

```diff
@@ -108,3 +109,84 @@ def threshold_mask(images, threshold, mask=None):
         if len(bad_pixels[0]) != 0:
             mask[bad_pixels] = 0
         yield mask
+
+
+def mask_edge(img_shape, edge_size):
```

Comment:
This is reasonable convenience function to have, but would you indulge some an opinion about the name of it? The name `mask_edge` makes me think it operates on the image itself and returns a masked copy. Maybe `margin_mask` or `draw_margin_mask`?